1.1.0/shipment-status.md

```diff
@@ -133,7 +133,7 @@ If all goes well, the company **Fast** will receive a response like this:
         }
       ],
       "shipmentStatus": "Active",
-      "shipmentArrivalStatus": "OnTime",
```

Comment:
The change on line #136 of `1.1.0/shipment-status.md` needs to be synchronized with the mock file `1.1.0/mock/samples/shipment-use-case.A.step-2.json`.